iati/core/resources.py

```diff
@@ -26,22 +26,25 @@
 
 BASE_PATH = 'resources'
 """The relative location of the resources folder."""
-BASE_PATH_CODELISTS = os.sep.join((BASE_PATH, 'codelists'))
-"""The relative location of the folder containing codelists from the SSOT."""
-BASE_PATH_CODELISTS_EMBEDDED = os.sep.join((BASE_PATH_CODELISTS, 'embedded'))
-"""The relative location of the folder containing embedded codelists from the SSOT."""
-BASE_PATH_CODELISTS_NON_EMBEDDED = os.sep.join((BASE_PATH_CODELISTS, 'non_embedded'))
-"""The relative location of the folder containing non-embedded codelists from the SSOT."""
-BASE_PATH_DATA = os.sep.join((BASE_PATH, 'data'))
+BASE_PATH_202 = os.sep.join((BASE_PATH, '202'))
+"""The relative location of the resources folder for version 2.02 of the IATI Standard."""
+PATH_CODELISTS = 'codelists'
+"""The location of the folder containing codelists from the SSOT."""
+PATH_CODELISTS_EMBEDDED = os.sep.join((PATH_CODELISTS, 'embedded'))
```

Comment:
ie. changing `embedded` to `IATI-Codelists`?

If so, there are a few problems (all small, but probably worth noting) with straight-up using that:

* It's mixed case, which could cause some problems depending on the underlying file system.
* The names and organisation of the git repos are a bit not nice, and use `IATI` more than they probably should.
* It's inconsistent with the folder names for Schemas / Rulesets (unless they're changed too).